src/Query/index.ts

```diff
@@ -187,6 +187,21 @@ export class Query {
     };
     const searchPropertyNames = Object.keys(searchPropertyMapping);
 
+    // DOP-3976: phrases found in conjunction should have additional score boost if they are found in order
```

Comment:
Should we add a `slop` value to this operator? Currently on the staging link, the results for, for example, "delete collection" and "collection delete" are quite different. Would it be worth adding a little bit of a `slop` value so that there's a bit of flexibility in how the query is asked?

Reply:
this can be a follow up but i think this will introduce more changes than the ask, in terms of ranking. this is specifically for the phrase asked without reordering.

but you bring up a good point - perhaps the boost is too large, as it will look for this phrase and boost for its existence - regardless of its content. will lower this boost while trying to keep the same result as described in the ticket

Reply:
@mayaraman19 added a slop factor for this and modified the boost! upon testing, this was pretty significant in boosting the phrase as is, and also preserving cases where order does not matter as in your example.